EdvClientCache.js

```diff
@@ -1,11 +1,12 @@
 /*!
  * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
+
+import LRU from 'lru-cache';
 
 export default class EdvClientCache {
```

Comment:
Is this EdvClientCache thing even used? If not, let's remove it now while we're here.

Reply:
No, removed.